low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+from cortx.sspl.bin.sspl_constants import PRODUCT_FAMILY
+from cortx.utils.process import SimpleProcess
+from cortx.utils.validator.v_pkg import PkgV
+from cortx.sspl.bin.error import SetupError
+
+TEST_DIR = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test"
+
+class SSPLTestCmd:
+    """Starts test based on plan (sanity | alerts)."""
+
+    def __init__(self, args: list):
+        self.args = args
+        self.name = "sspl_test"
+
+    def process(self):
+        CMD = f"{TEST_DIR}/run_tests.sh test {' '.join(self.args)}"
+        output, error, returncode = SimpleProcess(CMD).run()
```

Comment:
@gargpalak048 , please add TODO: Need to convert run_tests.sh from shell to python.

Reply:
Subprocess hides running test information(pass/fail) on the screen. We are validating script execution only here. 
cc: @cdeshmukh

Reply:
@gargpalak048 , Can you set **found_test_failure** flag inside sspl_test/run_test.py ?
At the time of script completion throw an custom exception if found_test_failure is True.
That way we can know test status. But this is just a temporary workaround only.

Reply:
Added the code for realtime output in process.py script